client/src/main/scala/skuber/api/package.scala

```diff
@@ -769,7 +770,7 @@ package object client {
 
     val sslContext = TLS.establishSSLContext(k8sContext)
     sslContext foreach { ssl =>
-      val httpsContext = ConnectionContext.https(ssl, None,Some(scala.collection.immutable.Seq("TLSv1.2", "TLSv1")), None, None)
+      val httpsContext = ConnectionContext.https(ssl, Some(AkkaSSLConfig()))
```

Comment:
Right now Skuber discards AkkaSslConfigs. I think there's value in it, for example in tests I needed to disable hostname verification which now I can.

Reply:
Agreed